node/node.go

```diff
@@ -660,11 +663,11 @@ func (n *Node) initMetricsAPI() error {
 	n.Log.Info("initializing metrics API")
 
 	dbNamespace := fmt.Sprintf("%s_db", constants.PlatformName)
-	db, err := meterdb.New(dbNamespace, registry, n.DB)
+	meterDBManager, err := n.DBManager.NewMeterDBManager(dbNamespace, registry)
```

Comment:
Should we do this in the database initialization? Feels kind of weird to assign to `n.DBManager` inside of `initMetricsAPI`

Reply:
This is done here so that it will only create the meterdb if the metrics API is enabled

Reply:
We could check if it's enabled in initDB, but it seems cleaner to handle here